src/Components/common/getUrlInputFile.js

```diff
@@ -0,0 +1,12 @@
+
+export const UrlInput = (e,setUrl,setFieldValue) =>{
```

Comment:
I think that this have more than one responsability but im not sure because its not clear what is being done here.